.travis.yml

```diff
@@ -36,7 +36,7 @@ install:
     - minikube update-context
 
     # Install Pulumi
-    - curl -L https://get.pulumi.com/ | bash -s -- --version 0.14.4-dev-1533786685-g80569fb5
+    - curl -L https://get.pulumi.com/ | bash -s -- --version 0.15.1-dev-1533943101-g58a75cbb
```

Comment:
Because we started to work on 0.15.0 yesterday, master of pulumi got tagged with 0.15.1.  This is just HEAD as of a few moments ago, and everything here will make it's way in to 0.15.0-rc2 and the final build.